### PR TITLE
Move __version__ out of _version as it's auto generated

### PR DESCRIPTION
**Pull request by Prepultrue**
Move `__version__` out of `_version.py` as it's automatically generated on build

### CODE REVIEW

File: arcana/core/utils.py [hunk 1]
```diff
@@ -16,11 +16,15 @@
 from collections.abc import Iterable
 import logging
 import attr
-from arcana._version import __version__
 from pydra.engine.task import FunctionTask
 from pydra.engine.specs import BaseSpec, SpecInfo
 from arcana.exceptions import ArcanaUsageError, ArcanaNameError, ArcanaVersionError
 
+# Avoid arcana.__version__ causing a circular import
```
Comment (tclose) [inline arcana/core/utils.py:23]: Why is this necessary? Where does the circular import come from?

**Comment by tclose** [comment]
Thanks @Prepultrue !

File: arcana/core/utils.py (hunk 1 above)
Comment (Prepultrue) [reply arcana/core/utils.py:23]: If the original import was also changed to
```python
from arcana import __version__
```
it would result in a circular import

Comment (tclose) [reply arcana/core/utils.py:23]: ok, I see
